specs/TrackingPrevention.md

```diff
@@ -0,0 +1,205 @@
+Tracking Prevention
+===
+
+# Background
+The WebView2 team has been asked for an API to toggle tracking prevention and also ability to control levels
+of tracking prevention.
+
+We are proposing two API's
+IsTrackingPreventionEnabled: This API allows you to enable/disable the tracking prevention feature. By default
+tracking prevention feature is enabled and set to `Balanced` level for WebView2. You can set the
+`CoreWebView2EnvironmentOptions.IsTrackingPreventionEnabled` property to false to disable the tracking prevention
+feature for WebView2 before creating environment that also skips the related code and improves the performance.
+
+TrackingPreventionLevel: This API allows you to control levels of tracking prevention for WebView2 which
+are associated with a profile when the feature is enabled and otherwise TrackingPreventionLevel is
+CoreWebView2TrackingPreventionLevel.Off and can't be changed to other kinds.
+The levels are similar to Edge: `Off`, `Basic`, `Balanced` and `Strict`.
+

```

Comment:
Does the Profile.TrackingPreventionLevel persist in the profile? Or do I have to reset it each time?

Reply:
Discussed: Yes it does. We need to ensure that this is documented in the ref docs as well.